packages/hmr-plugin/src/hmr-manager.ts

```diff
@@ -1,13 +1,13 @@
 import { ApplicationRef, ComponentRef, NgModuleRef } from '@angular/core';
 import { InitialState, NgxsBootstrapper } from '@ngxs/store/internals';
-import { createNewHosts, removeNgStyles } from '@angularclass/hmr';
 
 import { HmrStorage } from './internal/hmr-storage';
 import {
   BootstrapModuleFn,
   NgxsHmrLifeCycle,
   NgxsHmrOptions,
-  NgxsHmrSnapshot
+  NgxsHmrSnapshot,
+  OldHostRemoverFn
```

Comment:
The `OldHostRemoverFn` type should rather be defined in this file (and not exported) because its' only usages are within this file. We should keep types and interfaces close to their usage until they are needed in multiple places.

Reply:
It is very unfortunate that you take this approach in your applications, in normal languages such as Java, all interfaces should be taken out in a separate file, since in any case you do not make them public for other packages.

But anyway, I will make it do as you want.

Reply:
done

Reply:
I really do appreciate you adjusting your typical approach. I come from over a decade of C# and those typical statically typed approaches. There is a paradigm shift in the way that types and interfaces are used in TypeScript. In this case this type is used as an internal clarification of functionality, not an extensibility point for polymorphism.